src/bluetooth-fw/nimble/init.c

```diff
@@ -62,11 +62,6 @@ static void prv_host_task_main(void *unused) {
 
   ble_hs_cfg.sync_cb = sync_cb;
   ble_hs_cfg.reset_cb = reset_cb;
-  ble_hs_cfg.sm_our_key_dist |= BLE_SM_PAIR_KEY_DIST_ENC | BLE_SM_PAIR_KEY_DIST_ID;
```

Comment:
Did you test bonding still works with these removed?

Reply:
it is documented, https://github.com/pebble-dev/pebble-firmware/blob/main/third_party/nimble/syscfg/README.md

Reply:
Ah nice, sorry I'd forgotten it was there.